PyInstaller/utils/hooks/__init__.py

```diff
@@ -651,7 +651,7 @@ def is_module_or_submodule(name, mod_or_submod):
 PY_DYLIB_PATTERNS = [
     '*.dll',
     '*.dylib',
-    'lib*.so',
+    '*.so',
```

Comment:
Why is this necessary? This is a deep change, affecting a lot of things and should not be necessary for a hook.